plotly_resampler/figure_resampler/figure_resampler.py

```diff
@@ -130,6 +157,8 @@ def show_dash(
             See more https://dash.plotly.com/dash-core-components/graph
         **kwargs: dict
             Additional app.run_server() kwargs. e.g.: port
+            Note that these kwargs take precedence over the ones passed to the
```

Comment:
would add a line-break between the inline Note and the *e.g.: port*, as it looks a little too clumped now
![image](https://user-images.githubusercontent.com/38005924/178301738-e5473850-be2b-4e11-94fc-246e70814b69.png)